docs/components/squad.rst

```diff
@@ -118,6 +118,34 @@ Leadearboad <https://rajpurkar.github.io/SQuAD-explorer/>`__.
 .. _`QANet`: https://arxiv.org/abs/1804.09541
 .. _`BiDAF + Self Attention + ELMo`: https://arxiv.org/abs/1802.05365
 
+SQuAD with contexts without correct answers
+~~~~~
```

Comment:
Did you try to build the docs? Please, do 'make clean && make html' before every push!

Reply:
![image](https://user-images.githubusercontent.com/9271630/46876417-eaf8b300-ce46-11e8-9406-56897bdf6831.png)
It works!